### PR TITLE
chore: use GH actions

### PR DESCRIPTION
**Pull request by hugoArregui**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
+FROM node:16-alpine as base
```
Comment (marianogoldman) [inline Dockerfile:1]:
```suggestion
FROM node:18-alpine as base
```

Comment (hugoArregui) [reply Dockerfile:1]: I would do this as part of this PR, but I agree we should update this

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+jobs:
+  validations:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```
Comment (marianogoldman) [inline .github/workflows/build.yml:13]:
```suggestion
      - uses: actions/checkout@v4
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
```
Comment (marianogoldman) [inline .github/workflows/build.yml:15]:
```suggestion
        uses: actions/setup-node@v4
```

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+          POSTGRES_PASSWORD: 12345678
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v3
```
Comment (marianogoldman) [inline .github/workflows/build.yml:43]:
```suggestion
      - uses: actions/checkout@v4
```

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      - name: Use Node.js 16.16.0
+        uses: actions/setup-node@v3
```
Comment (marianogoldman) [inline .github/workflows/build.yml:45]:
```suggestion
        uses: actions/setup-node@v4
```

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        with:
+          node-version: 16.16.0
```
Comment (marianogoldman) [inline .github/workflows/build.yml:47]: Doesn't this work with node 18?

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**